app/src/chat_engine.py

```diff
@@ -17,6 +17,7 @@
 class OnMessageResult:
     response: str
     chunks: Sequence[Chunk]
```

Comment:
To simplify, we can always use `retrieve_with_scores()` instead of `retrieve()` and store this:
```suggestion
    chunks_with_scores: dict[Chunk, float]
```
cc: @KevinJBoyer

Reply:
In [my PR, I end up calling](https://github.com/navapbc/labs-decision-support-tool/pull/29/files#diff-576e93798fc73bf515ae76fa5bed4c8df408fc09b7e159cd95646cf12dd7deaaR20) `retrieve_with_scores()` within `retrieve()` anyway.

Reply:
I wonder if it doesn't make sense to remove `retrieve` entirely and just have `retrieve_with_scores` 🤔

Reply:
I'm thinking of going with the approach for removing `retrieve` entirely for more predictably